include/aruku/config/grpc/call_data.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef __ARUKU__CONFIG__GRPC__CALL_DATA_HPP__
+#define __ARUKU__CONFIG__GRPC__CALL_DATA_HPP__
+
+#include "aruku_interfaces/aruku.grpc.pb.h"
+#include "aruku_interfaces/aruku.pb.h"
+#include "aruku/config/grpc/call_data_base.hpp"
+#include "grpc/support/log.h"
+#include "grpcpp/grpcpp.h"
+
+namespace aruku
+{
+template <class ConfigRequest, class ConfigReply>
+class CallData : public CallDataBase
+{
+public:
+  CallData(
+    aruku_interfaces::proto::Config::AsyncService * service, grpc::ServerCompletionQueue * cq,
+    const std::string path);
```

Comment:
should be `const std::string &`.

```suggestion
    const std::string& path);
```

Please check other lines, remember this rule:
- Pass constant reference (`const &`) whenever it is possible.